lib/keycloak.js

```diff
@@ -129,9 +129,7 @@ function Keycloak (config) {
             kc.flow = initOptions.flow;
         }
 
-        if (initOptions.timeSkew != null) {
-            kc.timeSkew = initOptions.timeSkew;
-        }
+        kc.timeSkew = initOptions.timeSkew ?? 60;
```

Comment:
Since adding a default value ensures that the value is always set, the following code can be removed:
https://github.com/keycloak/keycloak-js/blob/7f5df13236b487d92a44f0155aacf6a6cea3ee5f/lib/keycloak.js#L606-L609

Additionally, the following `if` statement can be removed, leaving the body:
https://github.com/keycloak/keycloak-js/blob/7f5df13236b487d92a44f0155aacf6a6cea3ee5f/lib/keycloak.js#L990-L1002